src/main/webapp/app/shared/components/organisms/dynamic-table/dynamic-table.component.html

```diff
@@ -0,0 +1,52 @@
+<p-table
+  [value]="data()"
+  [paginator]="true"
+  [rowsPerPageOptions]="[10, 20, 50]"
+  [totalRecords]="totalRecords()"
+  [scrollable]="true"
+  scrollHeight="25rem"
+  [rows]="rows()"
+  [lazy]="true"
+  (onLazyLoad)="emitLazy($event)"
+  [loading]="loading()"
+  [autoLayout]="true"
+  [tableStyle]="{ width: 'auto' }"
+>
+  <!-- Header -->
+  <ng-template pTemplate="header">
+    <tr>
+      @if (selectAble()) {
```

Comment:
same comment here for "selectable"
```suggestion
      @if (selectable()) {
```